library/sync-tasks/src/androidTest/java/org/cru/godtools/sync/task/BaseDataSyncTasksTest.java

```diff
@@ -44,7 +44,7 @@ public void verifyStoreLanguageChangingCode() {
 
         // run test
         mTasks.storeLanguage(mEvents, language(1L, newLocale, false));
-        assertThat(mDao.find(Language.class, originalLocale), nullValue());
+        assertThat(mDao.find(Language.class, originalLocale, nullValue());
```

Comment:
oops... accidentally deleted one too many parentheses

```suggestion
        assertThat(mDao.find(Language.class, originalLocale), nullValue());
```